tests/lib/Compliance/HistoryComplianceTest.php

```diff
@@ -34,7 +34,7 @@
 use OCP\IUserSession;
 use OCP\Security\IHasher;
 use PHPUnit\Framework\MockObject\MockObject;
-use Test\TestCase;
+use PHPUnit\Framework\TestCase;
```

Comment:
Any reason for this change?
It's recommended to use https://github.com/nextcloud/server/blob/master/tests/lib/TestCase.php

Reply:
Hmm, honestly because i didn't know about and it didn't work with the Test\TestCase.
But then i'll revert that, there must be some other solution then.

Reply:
Is nextcloud/server probably still fixed to phpunit 8? 🤔

Reply:
I fixed the base class in the meantime in master of server

Reply:
Yeah, have seen that. Thanks! ☺️
It also works here, but i just dont get phpunit 9 to run on my local machine... 🤔